ftp/OlsonJJ/recuerdos-luna/recuerdos-luna.ly

```diff
@@ -0,0 +1,261 @@
+\version "2.18.2"
+
+\header {
+ title = "Recuerdos de la Luna"
+ subtitle = \markup { \center-column { { "Un arreglo sencillo de “Recuerdos de la Alhambra” por Francisco Tárrega" } { "en homenaje a “Claro de Luna” por Beethoven" } } }
+ %composer = "Arreglo:  Jeffrey Olson"
+ %arranger = "Dedos:  Dennis Burns"
+ composer = ##f
+ arranger = \markup { \abs-fontsize #10 \right-column { { "Arreglo:  Jeffrey Olson" } { "Dedos:  Dennis Burns" } } }
+ myfoot = ##f
+ style = "Romantic"
+ date = "1802,1896,2017" % (by Beethoven, Tárrega, Olson)
+ mutopiadate = "1896"
+ source = "Madrid: Orfeo Tracio, n.d. (ca.1920). Plate 1102" % via IMSLP24052
+ mutopiasource = "Madrid: Orfeo Tracio, n.d. (ca.1920). Plate 1102"
+ moreInfo = "This simple arrangement combines Tárrega's “Recuerdos de la Alhambra” 
+            with the triplets of Beethoven's “Moonlight Sonata”.  It was originally developed
+            as one of many sight reading exercises in “Chromatic Fretboard Exercises” to be
+            playable in a single fixed position (either VII or IX), but it sounds best
+            with the indicated fingering by Dennis Burns which allows each triplet arpeggio
+            to ring freely." 
+
```

Comment:
This needs to be one continuous line or it will fail to parse by the software that processes this line into a catalog entry. I understand that this can be difficult for some editors.